client/quantum_serverless/core/job.py

```diff
@@ -473,7 +473,7 @@ def __init__(
 
     def status(self):
         """Returns status of the job."""
-        return self._job_client.status(self.job_id)
+        return _map_status_to_serverless(self._job_client.status(self.job_id))
```

Comment:
We will do the translation from any supported job client status to Qiskit terminology here in the user-facing `Job` client. This is the only place we'll do the translation. Everything underneath this will be Ray terminology.